oak_functions/proto/unary_server.proto

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.session.unary.v1;
+
+option java_multiple_files = true;
+option java_package = "oak.session.unary.v1";
+
+message UnaryRequest {
+  bytes body = 1;
+  // Links a sequence of messages, declaring them as part of a single a session.
+  fixed64 session_id = 2;
+}
+
+message UnaryResponse {
+  bytes body = 1;
+}
+
+service UnarySession {
```

Comment:
Maybe add comments here and on the StreamingSession to explain how they are related to each other?